src/standalone/Soil/rre.jl

```diff
@@ -400,6 +411,7 @@ function ClimaLSM.make_update_jacobian(model::RichardsModel{FT}) where {FT}
             ),
         )
         # Hardcoded for single column: FIX!
+        # The derivative of the tendency should live in boundary vars and be updated there
```

Comment:
I'm confused - aren't we calculating the derivative of the tendency here (in ∂T_bc∂YN )? are you saying that should get stored in something like `p.bc.dT`?

Reply:
yeah...I was thinking eventually when we have the implicit solver working for energy/hydrology and know how itll work for integrated models, we might want to allocate space for this and update it as part of `update_boundary_vars!`...but Im not sure. more of a note for the future!